deploy/operator/operator.yaml

```diff
@@ -26,8 +26,6 @@ spec:
           effect: NoSchedule
       containers:
         - name: nmstate-operator
-          args:
-          - --v=production
```

Comment:
This is a serious regression. Without this flag, we'd flood logs with all the "Reconciled Node" logs. On a production system, that may bury relevant information about failures that happened a while back. Please propose an alternative or we may need to drop all the logs we find slightly irrelevant (like the periodic ones).

Reply:
Resurrected